utils_nlp/eval/compute_rouge.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import os
+import shutil
+import time
+import tempfile
+
+from pyrouge import Rouge155
+from rouge import Rouge
+from .rouge_ext import RougeExt
+
+
+def compute_rouge_perl(cand, ref, input_files=False):
```

Comment:
use is_input_files as a better name

Reply:
How about input_is_files?

Reply:
in python, i think the convention is to use is*, like isnumeric().

Reply:
OK. That makes sense. Changed it to is_input_files

Reply:
OK. That makes sense. Changed it to is_input_files